packages/block-library/src/table/edit.js

```diff
@@ -414,31 +414,35 @@ function TableEdit( {
 							},
 							columnIndex
 						) => (
-							<RichText
-								tagName={ CellTag }
-								key={ columnIndex }
-								className={ classnames(
-									{
-										[ `has-text-align-${ align }` ]: align,
-									},
-									'wp-block-table__cell-content'
-								) }
-								scope={ CellTag === 'th' ? scope : undefined }
-								colSpan={ colspan }
-								rowSpan={ rowspan }
-								value={ content }
-								onChange={ onChange }
-								onFocus={ () => {
-									setSelectedCell( {
-										sectionName: name,
-										rowIndex,
-										columnIndex,
-										type: 'cell',
-									} );
-								} }
-								aria-label={ cellAriaLabel[ name ] }
-								placeholder={ placeholder[ name ] }
-							/>
+							<CellTag key={ columnIndex }>
+								<RichText
+									key={ columnIndex }
+									className={ classnames(
+										{
+											[ `has-text-align-${ align }` ]:
+												align,
+										},
+										'wp-block-table__cell-content'
+									) }
+									scope={
+										CellTag === 'th' ? scope : undefined
+									}
+									colSpan={ colspan }
+									rowSpan={ rowspan }
```

Comment:
I think the `colSpan` and `rowSpan` props should be moved to the `CellTag` component. Otherwise, I don't think `colSpan` and `rowSpan` will be maintained when the raw data of the table is pasted into the block editor.

Reply:
Done. 👍